modules/javafx.graphics/src/main/java/com/sun/glass/ui/headless/HeadlessApplication.java

```diff
@@ -0,0 +1,217 @@
+        throw new UnsupportedOperationException("Not supported yet.");
+    }
+
+    @Override
+    protected void staticCursor_setVisible(boolean visible) {
+        throw new UnsupportedOperationException("Not supported yet.");
+    }
+
+    @Override
+    protected Size staticCursor_getBestSize(int width, int height) {
+        return new Size(16, 16);
+    }
+
+    @Override
+    public Pixels createPixels(int width, int height, ByteBuffer data) {
+        return new HeadlessPixels(width, height, data);
+    }
+
+    @Override
+    public Pixels createPixels(int width, int height, ByteBuffer data, float scalex, float scaley) {
+        return new HeadlessPixels(width, height, data, scalex, scaley);
+    }
+
+    @Override
+    public Pixels createPixels(int width, int height, IntBuffer data) {
+        return new HeadlessPixels(width, height, data);
+    }
+
+    @Override
+    public Pixels createPixels(int width, int height, IntBuffer data, float scalex, float scaley) {
+        return new HeadlessPixels(width, height, data, scalex, scaley);
+    }
+
+    @Override
+    protected int staticPixels_getNativeFormat() {
+        return Pixels.Format.BYTE_BGRA_PRE;
+    }
+
+    @Override
+    public GlassRobot createRobot() {
+        this.activeRobot = new HeadlessRobot(this);
+        return this.activeRobot;
+    }
+
+    @Override
+    protected double staticScreen_getVideoRefreshPeriod() {
+        return 0.;
+    }
+
+    @Override
+    protected Screen[] staticScreen_getScreens() {
+        final int screenWidth = 1000;
+        final int screenHeight = 1000;
+        if (this.screens == null) {
+            float scaleX = 1.f;
+            float scaleY = 1.f;
+            String scale = System.getProperty("glass.gtk.uiScale");
+            if (scale != null && !scale.isBlank()) {
+                float scaleFloat = Float.parseFloat(scale);
+                scaleX = scaleFloat;
+                scaleY = scaleFloat;
+            }
+            Screen screen = new Screen(0, 32, 0, 0, screenWidth, screenHeight, 0, 0, screenWidth, screenHeight, 0, 0, screenWidth, screenHeight, 100, 100, 1f, 1f, scaleX, scaleY);
+            this.screens = new Screen[1];
+            this.screens[0] = screen;
+            this.frameBuffer = ByteBuffer.allocate(screen.getWidth() * screen.getHeight() * 4);
+        }
+        return this.screens;
+    }
+
+    @Override
+    public Timer createTimer(Runnable runnable) {
+        return new HeadlessTimer(runnable);
+    }
+
+    @Override
+    protected int staticTimer_getMinPeriod() {
+        return 0;
+    }
+
+    @Override
+    protected int staticTimer_getMaxPeriod() {
+        return 1_000_000;
+    }
+
+    @Override
+    protected CommonDialogs.FileChooserResult staticCommonDialogs_showFileChooser(Window owner, String folder, String filename, String title, int type, boolean multipleMode, CommonDialogs.ExtensionFilter[] extensionFilters, int defaultFilterIndex) {
```

Comment:
This is an extremely long line...

Reply:
changed to the same format as com.sun.glass.ui.Application